renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -435,12 +435,7 @@ __attribute__((visibility("default"))) EGLContext eglCreateContext(EGLDisplay di
   init.depthBits = value;
   eglhooks.eglGetConfigAttrib_real(display, config, EGL_STENCIL_SIZE, &value);
   init.stencilBits = value;
-// TODO: how to detect this?
-#if ENABLED(RDOC_ANDROID)
   init.isSRGB = 0;
```

Comment:
I just had another read of the ES 3.2 spec pdf, GL_SRGB8 is marked as not color-renderable (table 8.10), which is why it gives incomplete framebuffers. 

Ive added that to the comment. Bizarrely SRGB8_ALPHA8 is color-renderable, so I suppose we could check if EGL_BUFFER_SIZE was 32, and then set isSRGB=1.